arduino-core/test/processing/app/helpers/FileUtilsTest.java

```diff
@@ -0,0 +1,292 @@
+package processing.app.helpers;
+
+import static org.mockito.AdditionalMatchers.or;
+import static org.mockito.Matchers.*;
+
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.junit.rules.TemporaryFolder;
+import org.junit.rules.Timeout;
+import org.junit.runner.RunWith;
+
+import org.mockito.Matchers;
+
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+
```

Comment:
delete space..
Put the imports in alphabetical order of the 1st package name. so everything in `J` first and them all in `O` and static imports last